src/containers/products/ChronicleProductPage.js

```diff
@@ -5,30 +5,148 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import FooterSection from '../footer/FooterSectionWithPageLinks';
+import { Content, PageDescriptionWrapper } from '../../components/layout/IntroComponents';
+import {
+  ExtendedFeatureDescription,
+  ExtendedFeatureHeader,
+  ExtendedFeaturesContent,
+  FeatureShot,
+  Icon,
+  ProductOverviewDescriptionWrapper,
+  ProductOverviewWrapper,
+  ProductShot,
+  ProductStamp,
+} from './styled/StyledProductComponents';
+import { 
+  MEDIA_QUERY_LG,
+  MEDIA_QUERY_MD,
+  MEDIA_QUERY_TECH_SM
+   } from '../../core/style/Sizes';
+import { NEUTRALS } from '../../core/style/Colors';
+import { PageIntro } from '../../components/headers/PageHeaders';
+import ChronicleIcon from '../../assets/logos/placeholder-logo.svg';
+import FeatureImg1 from '../../assets/images/chronicle/chronicle_participants.png';
+import FeatureImg2 from '../../assets/images/chronicle/chronicle_data.png';
+import FeatureImg3 from '../../assets/images/chronicle/mobile_screenshot_inapp.png';
+import FooterSectionWithPageLinks from '../footer/FooterSectionWithPageLinks';
 import PageSection from '../../components/layout/NewPageSection';
+import ProductImg from '../../assets/images/chronicle/chronicle_dashboard.png';
```

Comment:
This screenshot has your name in it. I don't think it's a good idea to include that.

Reply:
Removed name.